ha/core/controllers/pcs/node_controller.py

```diff
@@ -114,7 +119,27 @@ def start(self, nodeid: str) -> dict:
             ([dict]): Return dictionary. {"status": "", "msg":""}
                 status: Succeeded, Failed, InProgress
         """
-        raise HAUnimplemented("This operation is not implemented.")
+        _res = self.nodes_status([nodeid])
+        if isinstance(_res, str):
+            _res = json.loads(_res)
+        _all_node_status = _res.get("msg")
+        _node_status = _all_node_status.get(nodeid)
```

Comment:
```
Check fail count and perform cleanup on failed resourced then wait and check again and log if still failcount exists and return failure
```

after node started 
sleep 10s 
and check `pcs resource failcount show`
if it has No failcounts output then return output else run 
`pcs resource cleanup`
and return output

it will be better if this logic part of pcs_contrller function

Reply:
done